spark/src/test/scala/org/apache/comet/CometExpressionSuite.scala

```diff
@@ -1635,7 +1637,7 @@ class CometExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelper {
         case (Some(sparkException), Some(cometException)) =>
           assert(sparkException.getMessage.contains(dtype + " overflow"))
           assert(cometException.getMessage.contains(dtype + " overflow"))
-        case (None, None) => assert(true) // got same outputs
+        case (None, None) => checkSparkAnswerAndOperator(sql(query))
```

Comment:
nit: we call `checkSparkMaybeThrows,`, and if that doesn't fail, then we call `checkSparkAnswerAndOperator`. I think this could all be streamlined, but this is beyond the scope of this PR